### PR TITLE
fix(core-utils/query-params): don’t allow optimizing transfers

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
`optimize=TRANSFERS` seems to cause problems even on OTP1. This PR removes the dropdown from OTP1 instances, as it was already removed from OTP2 instances.

### CODE REVIEW

**Comment by miles-grant-ibigroup** [comment]
Good call binh! Some cruft removed

**Comment by miles-grant-ibigroup** [comment]
Good call binh! Some cruft removed

File: packages/trip-form/i18n/en-US.yml [hunk 1]
```diff
@@ -34,7 +34,6 @@ otpUi:
     optimizeBikeSpeed: Speed
     optimizeFor: Optimize for
     optimizeQuick: Speed
```
Comment (binh-dam-ibigroup) [inline packages/trip-form/i18n/en-US.yml:36]: These two strings should be removed too, they are for the "Optimize Walk" selector.

Comment (binh-dam-ibigroup) [reply packages/trip-form/i18n/en-US.yml:36]: Correction, only `optimizeQuick` should be removed here.

**Review by binh-dam-ibigroup — APPROVED** [review]
See https://github.com/opentripplanner/otp-ui/pull/393#discussion_r864112804 and it should be good to go.

**Review by philip-cline — APPROVED** [review]

**Pull request merged**